### PR TITLE
Add support for US SSN

### PR DESCRIPTION
**Pull request by DougGajda**
Used Regex to help search for SSN within the given sentence. Used the AssertEqual statement to check if the function and my list match up.

### CODE REVIEW

File: pii_team_frost.py [hunk 1]
```diff
@@ -22,7 +22,11 @@ def find_amex(text) -> list:
 def find_us_ssn(text) -> list:
     """Finds all occurrences of a US social security number in a text string"""
     # match a 9 digit social security number
-    return []
+    SSNRegex = re.compile(r'\d\d\d-\d\d-\d\d\d\d')
```
Comment (claesmk) [inline pii_team_frost.py:25]: This works, however, \d\d\d can be shortened to \d{3} - similar for the other \d patterns you have

File: pii_team_frost.py [hunk 2]
```diff
@@ -22,7 +22,11 @@ def find_amex(text) -> list:
...
+    MO = SSNRegex.search(text)
```
Comment (claesmk) [inline pii_team_frost.py:26]: I'll add some additional comments here later when I get a chance

**Comment by DrHinton410** [comment]
OK
________________________________

**Review by claesmk — CHANGES_REQUESTED** [review]
Please see embedded comments

**Comment by claesmk** [comment]
@DougGajda let me know if you have any questions

File: pii_team_frost.py [hunk 3]
```diff
@@ -22,7 +23,8 @@ def find_amex(text) -> list:
 def find_us_ssn(text) -> list:
     """Finds all occurrences of a US social security number in a text string"""
     # match a 9 digit social security number
-    return []
+    MO = re.findall(r'\d{3}-\d{2}-\d{4}', text)
+    return [MO]
```
Comment (claesmk) [inline pii_team_frost.py:27]: This is no longer a match object - what you were calling MO - so your variable name probably shouldn't be MO.  You are still trying to return that as a list.  I would
```
return re.findall(r'\d{3}-\d{2}-\d{4}', text)
```

**Review by claesmk — CHANGES_REQUESTED** [review]
Please see comments you are still returning a list within a list

**Review by claesmk — APPROVED** [review]
Looks like this is working OK, so will approve.  Not sure why you needed some extra imports
```
from pickle import FALSE
from unittest import result
from types import NoneType
```

**Pull request merged**